QUEST_LV_0100.tsv

```diff
@@ -980,7 +980,7 @@ QUEST_LV_0100_20150317_000975	{memo X}I hope that crazy magician, Antares, won't
 QUEST_LV_0100_20150317_000976	{memo X}He set a trap so well.{nl}I didn't like him even he was concious.
 QUEST_LV_0100_20150317_000977	{memo X}We have to be careful from here.{nl}This is the place where Antares, a dangerous magician, was imprisoned.{nl}We should move up to the next floor quickly.
 QUEST_LV_0100_20150317_000978	{memo X}He was a great magician before.{nl}But, his ideology was too dangerous.
-QUEST_LV_0100_20150317_000979	Damn, why now..{nl}
+QUEST_LV_0100_20150317_000979	Oh my goodness, why now..{nl}
```

Comment:
The original is more like "Oh my god!", as she is surprised that Antares is free. I wanted to change it to "Oh my Goddess!", but no idea if that would work. :yum: